src/App.tsx

```diff
@@ -1,24 +1,235 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, {
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import * as todoService from './api/todos';
+import { TodoRow } from './components/TodoRow';
+import { TodoHeader } from './components/TodoHeader';
+import { Status } from './types/Status';
+import { TodoFooter } from './components/TodoFooter';
+import { ErrorMessage } from './components/ErrorMessage';
+
+const getFilteredTodos = (todos: Todo[], isCompleted: Status): Todo[] => {
```

Comment:
Even better would be to move this function to separate file as a helper function